frontend/test/containers/containers.sagas.spec.ts

```diff
@@ -166,6 +217,32 @@ describe('Containers: sagas', () => {
 				.silentRun();
 		})
 	})
+
+	describe('updateContainerSettings', () => {
```

Comment:
You should perhaps add some error handling tests

Reply:
added error handling tests for both fetch and updated container settings